Fix explanation of asset installation and CSS usage (#852)
shopware/docs: guides/plugins/themes/add-assets-to-theme.md

{% endcode %}

> Next, please run the command `bin/console theme:compile`. The assets from the path defined in the `theme.json` file will be copied by the `theme:compile` command to `<shopware root>/public/theme/<theme-asset-uuid>` alongside with the compiled CSS and JS which are stored in a separate folder.

Reviewer: ```suggestion
Next, run the command `bin/console theme:compile`. The assets from the path defined in the `theme.json` file will be copied by the `theme:compile` command to `<shopware root>/public/theme/<theme-asset-uuid>` along with the compiled CSS and JS, which are stored in a separate folder.
```

{% endcode %}

Next, please run the commands `bin/console assets:install` and `bin/console theme:compile`. `assets:install` deletes the content of `<shopware root>/public/bundles/<your-theme-name>` and copies files from `<plugin root>/src/Resources/public` into it. The assets from the path defined in the `theme.json` file will be copied by the `theme:compile` to `<shopware root>/public/bundles/<your-theme-name>` and `<shopware root>/public/theme/<theme-uuid>` :

{% code title="<shopware root>/public" %}

```text
# 
.

> ├── bundles
> │   ├── framework
> │   ├── administration
> │   ├── storefront
> │   └── <your-plugin-name> <-- e.g. swagbasicexampletheme
> │       └── assets <-- Your asset is copied here
> │           └── your-image.png
> └── theme
>     └── <theme-uuid>
>         └── assets <-- Your asset is copied here
>             └── your-image.png

Reviewer 2: ```suggestion
└── theme
    ├── <theme-uuid>
    │   ├── css
    │   │   └── all.css
    │   └── js
    │       └── all.js
    └── <theme-asset-uuid>
        └── asset
            └── your-image.png   
```

You can link to the asset with the twig [asset](https://symfony.com/doc/current/templates.html#linking-to-css-javascript-and-image-assets) function:

```markup
<img src="{{ asset('/assets/your-image.png', 'theme') }}">
```

> In SCSS you can link to the asset like the following:

Reviewer: ```suggestion
In SCSS, you can link to the asset like the following:
```

You can link to the asset with the twig [asset](https://symfony.com/doc/current/templates.html#linking-to-css-javascript-and-image-assets) function:

```markup

> <img src="{{ asset('bundles/<your-plugin-name>/assets/your-image.png', 'asset') }}">

Reviewer 2: ```suggestion
<img src="{{ asset('/assets/your-image.png', 'theme') }}">
```

```

In SCSS you can link to the asset like the following:

```css
body {

>     background-image: url("../assets/your-image.png");

Reviewer 2: ```suggestion
    background-image: url('#{$app-css-relative-asset-path}/your-image.png');
```
